docs/how-to-guides/others/planning-evaluation-using-scenarios.md

```diff
@@ -0,0 +1,297 @@
+   this, define an action that starts the bicycle moving when the distance condition is satisfied.
+
+   ![scenario-create-tutorial-02.png](images%2Fplanning-evaluation-using-scenarios%2Fscenario-create-tutorial-02.png)
+
+5. As shown above, define a start condition for an action named `act_start_bicycle`. This condition checks the distance
+   between the ego vehicle and the bicycle. If the distance is less than or equal to 10 meters, the action is triggered.
+
+   ![scenario-create-tutorial-03.png](images%2Fplanning-evaluation-using-scenarios%2Fscenario-create-tutorial-03.png)
+
+6. After the start condition, define an **event** for this action. Since there are no other conditions for this event,
+   set a dummy condition where SimulationTime is greater than 0, as shown above.
+
+   ![scenario-create-tutorial-04.png](images%2Fplanning-evaluation-using-scenarios%2Fscenario-create-tutorial-04.png)
+
+7. Define an **action** for this event. This action will set the velocity of the actor, Bicycle0, to 2 m/s.
+
+   ![scenario-create-tutorial-05.png](images%2Fplanning-evaluation-using-scenarios%2Fscenario-create-tutorial-05.png)
+
+8. The scenario is now ready to be tested. You can export the scenario from the `Scenario` tab by clicking the `Export`
+   button.
+
+   ![scenario-create-tutorial-06.png](images%2Fplanning-evaluation-using-scenarios%2Fscenario-create-tutorial-06.png)
+
+### Important Notes for Scenario Creation
+
+- The tutorial we made is a simple scenario implementation for more complex definitions and other road user actions,
+  please refer to
+  the [Scenario Simulator documentation](https://tier4.github.io/scenario_simulator_v2-docs/).
+- Due to the complexity of the scenario format, support for editing in the GUI is not yet complete for all scenario
+  features. Therefore, if there are any items that cannot be supported or are difficult to edit using the GUI, you can
+  edit and adjust the scenario directly using a text editor. In that case, the schema of the scenario format is checked
+  before saving, so you can edit with confidence.
+- Best way to understand other complex scenario implementations is to look at the existing scenarios. You can check the
+  scenarios in the
+  [Autoware Evaluator](https://evaluation.ci.tier4.jp/) to see other implementations.
+
+## Running Scenarios using Scenario Simulator
+
+### Creating the Scenario Simulator Environment
+
+- Clone the Autoware main repository.
+
+```bash
+git clone https://github.com/autowarefoundation/autoware.git
+```
+
+- Create source directory.
+
+```bash
+cd autoware
+mkdir src
+```
+
+- Import the necessary repositories both for the Autoware and Scenario Simulator.
+
+```bash
+vcs import src < autoware.repos
+vcs import src < simulator.repos
+```
+
+- If you are installing Autoware for the first time, you can automatically install the dependencies by using the
```

Comment:
In my humble opinion, it would be better refer here as well to the how to installation steps in autoware documentation if the user is first time using autoware.